hazelcast/src/main/java/com/hazelcast/map/impl/query/AggregationResult.java

```diff
@@ -55,10 +55,14 @@ public AggregationResult(Aggregator aggregator) {
 
     @Override
     public void combine(AggregationResult result) {
+        Collection<Integer> otherPartitionIds = result.getPartitionIds();
```

Comment:
maybe in `AggregationResult` class,  we can assign `partitionIds` to `emptyList()` to get rid of null checks.